mirgecom/eos.py

```diff
@@ -150,8 +186,14 @@ class MixtureEOS(GasEOS):
     .. automethod:: get_production_rates
     .. automethod:: species_enthalpies
     .. automethod:: get_species_source_terms
+    .. automethod:: get_temperature_seed
     """
 
+    @abstractmethod
+    def get_temperature_seed(self, cv: ConservedVars,
+                              temperature_seed: DOFArray = None):
```

Comment:
* Document return type
* Document `temperature_seed` type precisely
* Say what it's good for.

Reply:
(e4b9829)